samples/quickstart.js

```diff
@@ -13,14 +13,14 @@
 // limitations under the License.
 
 // [START gaxios_quickstart]
-const {request} = require('gaxios');
+const { request } = require('gaxios');
```

Comment:
So... how about disabling this option in prettier config to make the result much closer to what we have now?

```json
        "bracketSpacing": {
          "description": "Print spaces between brackets.",
          "default": true,
          "type": "boolean"
        }
```

Reply:
_nods_ we absolutely could do that.  I was thinking of reducing the amount of prettier customizations we do, and keep it as close to the default config as possible.  That includes dropping bracket spacing, and the trailing commas.

If we want to optimize for lowest changes, we could keep adding those customizations in.

Reply:
Oh. There is a serious risk of starting a discussion about style here :)

I get your point about keeping minimal config, but one thing about `gts` default style is that it more or less did what's explained in [Google JavaScript Style Guide](https://google.github.io/styleguide/jsguide.html). For some reason I feel it might make sense to adhere to that style guide :)  It puts no space between brackets, and uses trailing commas.

![image](https://user-images.githubusercontent.com/4015807/73720269-63d3ec00-46d6-11ea-9b8a-c5365907e8be.png)

![image](https://user-images.githubusercontent.com/4015807/73720303-7a7a4300-46d6-11ea-9e9e-06e7bc7cbe9e.png)

But again, I just _thought_ that keeping the formatting (if not too hard) is a good idea, but I'm OK with either decision.

Reply:
Sounds good :) gts update here:
https://github.com/google/gts/pull/440